internal/cmd/generate.go

```diff
@@ -418,6 +418,7 @@ func codegen(ctx context.Context, combo config.CombinedSettings, sql outPair, re
 	case sql.Gen.Go != nil:
 		out = combo.Go.Out
 		handler = ext.HandleFunc(golang.Generate)
+		sql.Gen.Go.Rename = combo.Rename
```

Comment:
I'm not certain this assignment is the best idea but it was the simplest change that worked. Combined settings should have the merged rename map from global and package levels.